crypto/hashtable/hashtable.c

```diff
@@ -639,8 +639,7 @@ HT_VALUE *ossl_ht_get(HT *h, HT_KEY *key)
         CRYPTO_atomic_load(&md->neighborhoods[neigh_idx].entries[j].hash,
```

Comment:
Shall we address this as well?

Reply:
The hash is uint64_t. There is nothing to address here.